app/channels/game_channel.rb

```diff
@@ -19,6 +19,11 @@ def subscribed
   end
 
   def flee
+    unless current_user.player == @gameboard.current_player
+      PlayerChannel.broadcast_to(current_user, { type: 'ERROR', params: { message: "It's not your round, you can't flee..." } })
+      return
```

Comment:
PlayerChannel.broadcast_error(current_user, "You can't fight against this card!")
gibt a bestehende methode fürs error broadcasten